Core/detection.js

```diff
@@ -79,29 +79,16 @@
 
         var xhr = XMLHttpRequest.prototype
         var originalOpen = xhr.open
-        var originalSend = xhr.send
 
         xhr.open = function(method, url) {
```

Comment:
I'll let Dave review this as he's the js expert, I'll jump in and test it once it is reviewed. One question that goes beyond this PR, do we need to cover other xhr open methods https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/open?

Reply:
Good point. It's actually a single function with variable args, but we should use the arguments array directly, rather than reconstructing it. I'll update that.